src/assets/css/style.css

```diff
@@ -18,6 +18,7 @@
 @import "pages/organization.css";
 @import "pages/about.css";
 @import "pages/agreement.css";
+@import "pages/impressum.css";
```

Comment:
For a future PR, but why would we have page specific CSS be included on every page?